distribution/docs/src/main/jdocs/content/_eventing/subscriptions-contents.adoc

```diff
@@ -0,0 +1,63 @@
+:title: Subscriptions
+:type: eventing
+:status: published
+
+==== Subscriptions
+
+Subscriptions represent "standing queries" in the Catalog.
+Like a query, subscriptions are based on the OGC Filter specification.
+
+===== Subscription Lifecycle
+
+A Subscription itself is a series of events during which various plugins or transformers can be called to process the subscription.
+
+====== Creation
+
+* Subscriptions are created directly with the <<_event_processor,Event Processor>> or declaratively through use of the Whiteboard Design Pattern.
+* The Event Processor will invoke each Pre-Subscription Plugin and, if the subscription is not rejected, the subscription will be activated.
+
+====== Evaluation
+
+* When a metacard matching the subscription is created, updated, or deleted in any Source, each Pre-Delivery Plugin will be invoked.
+
+* If the delivery is not rejected, the associated Delivery Method callback will be invoked.
+
+====== Update Evaluation
+
+Notably, the Catalog allows event evaluation on both the previous value (if available) and new value of a Metacard when an update occurs.
+
+====== Durability
+
+Subscription durability is not provided by the Event Processor.
+Thus, all subscriptions are transient and will not be recreated in the event of a system restart.
+It is the responsibility of Endpoints using subscriptions to persist and re-establish the subscription on startup.
+This decision was made for the sake of simplicity, flexibility, and the inability of the Event Processor to recreate a fully-configured Delivery Method without being overly restrictive.
+
+[IMPORTANT]
+====
+*Subscriptions are not persisted by the Catalog itself.* +
+Subscriptions must be explicitly persisted by an endpoint and are not persisted by the Catalog.
+The Catalog Framework, or more specifically the Event Processor itself, does not persist subscriptions.
+Certain endpoints, however, can persist the subscriptions on their own and recreate them on system startup.
+====
+
+==== Creating a Subscription
+
+Currently, the Catalog reference implementation does not contain a subscription endpoint.
+Nevertheless, an endpoint that exposes a web service interface to create, update, and delete subscriptions would provide a client's subscription's filtering criteria to be used by Catalog's Event Processor to determine which create, update, and delete events are of interest to the client.
```

Comment:
I'm not 100% sure, as that is a very convoluted sentence. Attempting to simplify.